README.md

```diff
@@ -1 +1,25 @@
-# microverse-m1w1-d2-project-production
\ No newline at end of file
+# HTML & CSS Linters Setup.
+
+This Microverse Project is built to deepen our knowledge on linters and Github workflow.
+
+
+## Built With
+
+- CSS HTML Git Command Language
+- Git Version Control System
+
+
+## Getting Started
+
+**Setup HTML & CSS Linters in Workflows folder through Actions ob Github**
+**Add .gitignore file**
+**Add CSS and HTML files file**
+**Complete test and add test.md file**
+
+
```

Comment:
Kindly add how to have a local copy of your project and run it. THis is what the getting started should contain.